Placeeo/Controller/MainViewController.swift

```diff
@@ -6,22 +6,30 @@
 //
 
 import UIKit
+import CoreLocation
+import MapKit
 
-class ViewController: UIViewController {
+class MainViewController: MapViewController{
 
-    var placesArray: [Place] = []
-  
-    
+  let mapVC = MapViewController()
     override func viewDidLoad() {
         
         super.viewDidLoad()
-        let mapViewController = MapViewController()
         
-        mapViewController.show(places: placesArray)
-       
-        present(mapViewController, animated: true, completion: nil)
+          let placeOne = Place(name: "Castle Uzhgorod", lat: 48.62216095313416, long: 22.308029690972464)
+          let placeTwo = Place(name: "Church", lat: 48.623380761219515, long: 22.302193204377218)
+          
+          
+          let places = [placeOne, placeTwo]
```

Comment:
where do you use these places?

Reply:
its for future using and creating MKAnottation

Reply:
I see. it's the right place for them, but let's not commit the things which are not used anywhere. you can have them locally and commit them later if needed

Reply:
check again!